data/json/techniques.json

```diff
@@ -1938,34 +1938,6 @@
     "defensive": true,
     "miss_recovery": true
   },
-  {
-    "type": "technique",
-    "id": "tec_muay_thai_rapid",
```

Comment:
So the issue here is that you've deleted a bunch of muay thai stuff that was added before the pr was reopened.  And that is breaking tests.

Reply:
@sadenar your best bet is copy the current version of `techniques.json`, paste it, and tweak the descriptions again